### PR TITLE
fix: Get rdkafka to build on windows

### LINKED ISSUE #230: Building rdkafka on Windows fails
[rdkafka](https://github.com/fede1024/rust-rdkafka) can't be reliably built on Windows at the moment, because it tries to run `./configure` as a build step and fails, not being able to find `bash`.
We should probably patch `build.rs` there to support MSVC.

### PR DESCRIPTION
**Pull request by Swatinem**
rdkafka-sys has a `cmake-build` feature that works OOTB on my windows.

fixes #230

#skip-changelog

### CODE REVIEW

File: relay-server/Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ processing = [
     "minidump",
     "rdkafka",
     "rdkafka-sys",
+    "rdkafka-sys/cmake-build",
```
Comment (jan-auer) [inline relay-server/Cargo.toml:20]: Do you know if it is possible to activate this only on windows?

Comment (Swatinem) [reply relay-server/Cargo.toml:20]: That would be a nightly feature:
https://github.com/rust-lang/cargo/issues/7914
(https://github.com/rust-lang/cargo/issues/2524)

Comment (jan-auer) [reply relay-server/Cargo.toml:20]: Gotcha, thanks. Let's add a note to the README then (where it says that we require latest rust), then this is good to merge 👍

**Review by jan-auer — APPROVED** [review]

**Comment by dashed** [comment]
Just a note. After this PR, I needed to do `brew install cmake` on macOS.

**Comment by jan-auer** [comment]
@dashed this has been added to the README in this PR, see:

https://github.com/getsentry/relay/pull/811/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R40

**Pull request merged**